File: Algorithm/Java/Remove Duplicates from Sorted Array.java
public class Solution {
    public int removeDuplicates(int[] A) {
        if (A == null || A.length == 0) {
            return 0;
        }
        int tail = 0;
        for (int i = 1; i < A.length; i++) {
            if (A[i] != A[i - 1]) {
                A[++tail] = A[i];
            }
        }
        return ++tail;
    }
}

File: Algorithm/JavaScript/Remove Duplicates from Sorted Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if (!nums.length) {
        return 0;
    }
    let index = 1;
    for (let i = 1; i < nums.length; ++i) {
        if (nums[i] === nums[i - 1]) {
            continue;
        }
        nums[index++] = nums[i];
    }
    return index;
};

console.log(removeDuplicates([0,0,1,1,1,2,2,3,3,4]));